### ISSUE TITLE
Broken links for lingads download

### ISSUE THREAD

**Issue by CarlosCaris** [janash/iqb-2024#3]
In the `EC_class_ligands_search_pc.ipynb` there is a variable called `baseUrl = "https://files.rcsb.org/ligands/download/"`, which is broken. Searching on the source website the new base url should be "https://files.rcsb.org/ligands/" and the construction of the download string could be as forme as follows:

```
baseUrl = "https://files.rcsb.org/ligands/"

for ChemID in molResultL:
    first_part_url = ChemID[0]
    second_part_url = ChemID
    cFile = f"{ChemID}_ideal.mol2"
    cFileUrl = baseUrl + first_part_url + "/" + second_part_url + "/" + cFile
    cFileLocal = "ligands/" + cFile
    response = requests.get(cFileUrl)
    with open(cFileLocal, "w+") as file:
        file.write(response.text)
```

**Comment by janash** [comment]
Hi @CarlosCaris - 
Thank you for filing this issue!

Taking a look at this, it actually seems to me that the mol2 file format is no longer supported.

This is the original URL: https://files.rcsb.org/ligands/download/11U_ideal.mol2

Checking the documentation page (https://www.rcsb.org/docs/programmatic-access/file-download-services) - it seems that `/download` will still work in the URL, but the file extension must be changed to SDF

https://files.rcsb.org/ligands/download/11U_ideal.mol2

I tried removing `/download` as you suggest, but also got a 404 for that.

**Comment by janash** [comment]
Closed by #4
